cfgov/jobmanager/migrations/0010_cleanup_unused_fields.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
```

Comment:
Not a huge deal but could this file (and the above) be named something more descriptive, like, `0010_remove_slug_and_active_fields` or `0010_jobs_cleanup_unused_fields`